case-lib/lib.sh

```diff
@@ -481,6 +502,16 @@ logger_disabled()
         return 0
     fi
 
+    ipc4_used && {
+        # TODO:
+        # Need to remove disabling sof-logger
+        # after sof-logger support for IPC4 has been provided in the future
+        dlogi 'Currenly sof-logger is not supported when running IPC4 mode'
```

Comment:
Currenly should be Currently. 

Sorry I didn't respond before the PR was merged.